Sources/Sharing/SharedKey.swift

```diff
@@ -90,12 +90,13 @@ extension Shared {
   ///   loading and saving the shared reference's value from some external source.
   public func load(_ key: some SharedKey<Value>) async throws {
     await MainActor.run {
-      reference.touch()
       @Dependency(PersistentReferences.self) var persistentReferences
-      reference = persistentReferences.value(
-        forKey: key,
-        default: wrappedValue,
-        skipInitialLoad: true
+      projectedValue = Shared(
```

Comment:
Instead of remembering to do the `reference.touch()` dance we can leverage `projectedValue =`.